gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/LocalGcsHelper.java

```diff
@@ -1,68 +0,0 @@
-/*
```

Comment:
Doing things this way prevents the FakeStorageRpc instance from being stored as a private final field in the test class. That's going to prevent us from having methods that allow us to dip directly into the "fake" storage maps.